tap_marketo/client.py

```diff
@@ -90,6 +94,11 @@ def __init__(self, endpoint, client_id, client_secret,
         self.job_timeout = job_timeout
         self.poll_interval = poll_interval
 
+        if request_timeout and float(request_timeout):
+            self.request_timeout = float(request_timeout)
+        else:
+            self.request_timeout = REQUEST_TIMEOUT # If value is 0,"0" or "" then set default to 300 seconds.
```

Comment:
Where ever there is a code change however small it is please add some comments

Reply:
Added comments